lib/parsers/asm-parser-ewavr.ts

```diff
@@ -138,13 +160,15 @@ export class AsmEWAVRParser extends AsmParser {
             if (matches) {
                 currentLabel = {
                     lines: [],
-                    initialLine: currentLine,
+                    initialLine: currentLine || 0,
```

Comment:
This change (and the other `|| 0` below I'm less sure of since I'm not familiar with ewavr. Should `currentLine` always be non-undefined here?

Reply:
I have no idea to be honest, was fighting the checker and the prior code.